test/infrastructure/container/docker.go

```diff
@@ -96,19 +96,19 @@ func (d *dockerRuntime) SaveContainerImage(ctx context.Context, image, dest stri
 // already exist. This is important when we're using locally build images in CI which
 // do not exist remotely.
 func (d *dockerRuntime) PullContainerImageIfNotExists(ctx context.Context, image string) error {
-	filters := dockerfilters.NewArgs()
-	filters.Add("reference", image)
-	images, err := d.dockerClient.ImageList(ctx, types.ImageListOptions{
-		Filters: filters,
-	})
+	imageExistsLocally, err := d.ImageExistsLocally(ctx, image)
 	if err != nil {
 		return fmt.Errorf("failure listing container images: %v", err)
```

Comment:
this doubles the error wrap now from `ImageExistsLocally` / adds the same string to the message.

Also: we should wrap the error.

Reply:
addressed!